pay-api/src/pay_api/resources/refund.py

```diff
@@ -0,0 +1,52 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Resource for Refunds endpoints."""
+from http import HTTPStatus
+
+from flask import current_app, jsonify, request
+from flask_restplus import Namespace, Resource, cors
+
+from pay_api.exceptions import error_to_response, BusinessException
+from pay_api.schemas import utils as schema_utils
+from pay_api.services import RefundService
+from pay_api.utils.auth import jwt as _jwt
+from pay_api.utils.errors import Error
+from pay_api.utils.util import cors_preflight
+
+API = Namespace('refunds', description='Payment System - Refunds')
+
+
+@cors_preflight('POST')
+@API.route('/refunds', methods=['POST', 'OPTIONS'])
+class Refund(Resource):
+    """Endpoint resource to create refunds against invoices."""
+
+    @staticmethod
+    @cors.crossdomain(origin='*')
+    @_jwt.requires_auth
```

Comment:
i thought we do refund by the service account ;rather than the user jwt.. ?

Reply:
That's a good catch.. I will make this change and update this PR.

Reply:
Done